### DISCUSSION TITLE
Update with Instructions for newer Sequent Microsystems components

### DISCUSSION THREAD

**Discussion by spifftek70** [tagyoureit/nodejs-poolController#1053] [Nixie]
The [Sequent Microsystems Building Automation Hat](https://sequentmicrosystems.com/products/raspberry-pi-building-automation) and [Sequent Microsystems 8 Relay Hat](https://sequentmicrosystems.com/collections/all-io-cards/products/raspberry-pi-relays-stackable-card) components listed in the install instructions for [DIY Standalone Nixie Pool Controller](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller#sequent-microsystems-8-relay-hat) are no longer sold and I'm guessing, have made obsolete by the manufacturer.  

Are there any plans to update the Wiki with instructions for the newer counterparts? Which I'm assuming are these:
[Eight Relays 4A/120V 8-Layer Stackable HAT for Raspberry Pi](https://sequentmicrosystems.com/collections/building-automation/products/8-relays-stackable-card-for-raspberry-pi)
[Building Automation V4 8-Layer Stackable HAT for Raspberry Pi](https://sequentmicrosystems.com/products/building-automation-8-layer-stackable-hat-v4-for-raspberry-pi?srsltid=AfmBOopTXKDG5uiyxPxaF_ArhwtnQegB2MErnHUiFLPmeWS1nj15arV-)

Also, is there any reason to keep using Raspberry Pi 4 rather than 5?

**Comment by tagyoureit** [comment]
I updated the links, but unsure who has used the v4 automation hat.  Unless Sequent updated their protocol (which they _are_known for doing), it should work fine.

There is no reason to upgrade your Pi.  The software worked fine on Pi3b and doesn't have new requirements since then.  The biggest change recently was to Bookworm and how it addresses the i/o ports but that was sorted out.  There's also no reason to not upgrade if you want.

**Reply by spifftek70** [reply]
Not really upgrading, as replacing ones that need it.  The Phoenix desert is a bit harsh on these components. 

I have the new Sequent stacks, but have no idea on the jumpers / DIP switch settings.  
Instead of two pair of RS 485 terms (4 slots), there are 5 slots, which is confusing.  The 8-Layer Stackable HAT also seems to handle RS 485 as well, but I'm not sure how.

Also, working on a 3d rendering of my [computer box](https://collaborate.shapr3d.com/v/jwS33AITi9jNqbSXuUl_N):

**Comment by spifftek70** [comment]
@tagyoureit 

updated instructions for the new Mega-Bas are [here](https://github.com/SequentMicrosystems/megabas-rpi/issues/20).
and [Here](https://github.com/SequentMicrosystems/megabas-rpi/wiki/User's-guide-4.2#errata)
Apparently there were some errors in the board printing process.
